classes/DataWarehouse/Visualization/TimeseriesChart.php

```diff
@@ -448,6 +448,9 @@ public function configure(
                                 $allNull = false;
                                 $visiblePoints++;
                             }
+                            if ($visiblePoints > 1) {
+                                break;
+                            }
                         }
                         if($allNull)
```

Comment:
```suggestion
                        if($visiblePoints > 0)
```
and can get rid of `$allNull` variable.

Reply:
I think it should be `if($visiblePoints == 0)` so that we know its "allNull" and can skip.

Reply:
You're right!